unocorn-squad/job-posting-app
# Create Job Details Card component

### Issue

Closes issues #14 and #22 

### Type of change

- [x] New Feature: a non-breaking change that adds new functionality.
- [x] Style: changes that do not affect the meaning of the code, e.g., white space, formatting, missing semicolons, etc.

### Summary

Create the job details component
- It's currently using the first item from the data.json file for the mock data
- I didn't style the button as I believe that's to be replaced with the actual Button component

### Testing steps

To test or see this component, import it into the app.jsx page for quick viewing until it has a page to reside 

### Relevant screenshots

![screencapture-127-0-0-1-5173-2024-07-10-19_57_08](https://github.com/unocorn-squad/job-posting-app/assets/16721486/423e7e4f-1aba-412c-8908-ab362521e3a6)
